Don't install the shasum program from this module as it's included in the Perl
distribution.
==========================================================================
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -7,7 +7,6 @@
 %att = (
 	'NAME'		=> 'Digest::SHA::PurePerl',
 	'VERSION_FROM'	=> $PM,
-	'EXE_FILES'	=> [ 'shasum' ],
 );
 
 my $MMversion = $ExtUtils::MakeMaker::VERSION || 0;
